webcompat/form.py

```diff
@@ -216,7 +223,7 @@ def build_formdata(form_object):
 **Steps to Reproduce**
 {description}
 
-'''.format(**formdata)
+""".format(**formdata)
```

Comment:
@cch5ng For now we are only planning to allow some people to edit the summary of an issue. 

----

@cch5ng I'm not sure if you are asking about `str.format()` or `**formdata` syntax. Let's assume it's about [`**kwargs`](https://docs.python.org/dev/tutorial/controlflow.html#unpacking-argument-lists)

`**` is an operator to unpack keyword arguments. An example:

```python
>>> # a dictionary of data
>>> formdata = {'state': 'California', 'city': 'Los Angeles'}
>>> # no keys in the string
>>> 'No luck'.format(**formdata)
'No luck'
>>> # Some parameters in the string matching the keys.
>>> '{city} is a city in {state}'.format(**formdata)
'Los Angeles is a city in California'
```